packages/dotnet-sdk/src/TeamsFx/Microsoft.TeamsFx.csproj

```diff
@@ -25,7 +25,6 @@
   </ItemGroup>
```

Comment:
Please also remove the line "AddRazorSupportForMvc", it should be related to auth page hosting.